api/cas-server-core-api-events/src/main/java/org/apereo/cas/support/events/authentication/CasAuthenticationPolicyFailureEvent.java

```diff
@@ -23,11 +25,14 @@ public class CasAuthenticationPolicyFailureEvent extends CasAuthenticationTransa
 
     private final Authentication authentication;
 
+    private final ClientInfoDTO clientInfoDTO;
```

Comment:
Probably should rename the class and variable without the DTO suffix, I don't think that convention is used anywhere else in CAS.

Reply:
@mmoayyed  changed the ClientInfo object to Serializable about 8 hours ago, so I can probably just remove the class altogether.  I'm wondering if should look for the released version of that library to use it or if that is going to be done at a later date.

Reply:
The release was done 8 hours ago :)

Reply:
Looking closer, I did see he bumped the version, so I'll go ahead and update it.

Reply:
This is now using the ClientInfo object.  I also reduced the number of threads produced to 20, and set the threadpool size to 10.